activestate.macos.yaml

```diff
@@ -0,0 +1,70 @@
+constants:
+  - name: SYSTRAY_APP
+    if: eq .OS.Name "MacOS"
+    value: ActiveState\ Desktop.app
+scripts:
+  - name: build-systray-app
+    language: bash
+    standalone: true
+    if: eq .OS.Name "MacOS"
+    description: Copies the MacOS application directory to the build dir
+    value: |
+      mkdir -p build/${constants.SYSTRAY_APP}
+
+      cp -r installers/${constants.SYSTRAY_APP} build/
+  - name: generate-update
+    language: bash
+    standalone: true
+    if: eq .OS.Name "MacOS"
+    description: Generate update files
+    value: |
+      set -e
+      $constants.SET_ENV
+
+      export GOARCH=${1:-amd64}
+      mkdir -p ./build/update
+      go run scripts/update-generator/main.go --systemAppDir $BUILD_TARGET_DIR/$constants.SYSTRAY_APP -o ./build/update \
```

Comment:
Isn't the default systemAppDir already correct? I don't think this runner needs anything macOS specific.

Reply:
I'm not following what the default systemAppDir is that you're referring to. The value being passed in here is the one copied from the `installers` directory to the build dir